modules/nextflow/src/main/groovy/nextflow/conda/CondaCache.groovy

```diff
@@ -60,6 +60,8 @@ class CondaCache {
 
     private String createOptions
 
+    private Boolean useMamba = false
```

Comment:
Minor, null is useless in this case better `private boolean useMamba` (false is default)

Reply:
Hmm, good to know!

I've refactored accordingly.